content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -553,7 +554,7 @@ plugins:
     limitedResources:
     - resource: pods
       matchScopes:
-      - scopeName: PriorityClass 
```

Comment:
Please omit whitespace fixes from this PR. It's OK to fix those in a separate PR.